### PR TITLE
Deleted unused variables from contracts-bedrock and add changeset

### PR DESCRIPTION
**Pull request by protocolwhisper**
**Description**

Deleted the unused Variables from the Typescript File

**Tests**

No test for deleting unused Variables

**Metadata**
- Fixes https://github.com/ethereum-optimism/optimism/issues/3838

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -95,6 +95,7 @@
   "dependencies": {
     "@changesets/cli": "^2.16.0",
     "@codechecks/client": "^0.1.11",
-    "@ethersproject/abstract-provider": "^5.7.0"
+    "@ethersproject/abstract-provider": "^5.7.0",
+    "changeset": "^0.2.6"
```
Comment (maurelian) [inline package.json:99]: Why is [this](https://www.npmjs.com/package/changeset) needed?

**Review by maurelian — CHANGES_REQUESTED** [review]
This PR would be OK without the changes to `package.json` and `yarn.lock`

File: package.json (hunk 1 above)
Comment (maurelian) [reply package.json]: hmmm, I think you must have run `yarn add changeset` to cause that diff. All good though, thanks.

Comment (protocolwhisper) [reply package.json]: You are right, my bad I did run `yarn add changeset`

**Comment by maurelian** [comment]
CI should be fixed by rebasing from develop.

File: package.json (hunk 1 above)
Comment (maurelian) [reply package.json]: https://github.com/ethereum-optimism/optimism/pull/4003#issuecomment-1315682790

Your changes look good, but we accidentally introduced a linter error to `develop` which is causing this branch to fail in CI. You should be able to fix this by rebasing your branch on develop. I believe these commands should achieve that:

```
git pull --rebase https://github.com/ethereum-optimism/optimism develop
git push --force
```

**Comment by protocolwhisper** [comment]
> CI should be fixed by rebasing from develop.

Successfully fetched and merged from upstream ethereum-optimism:develop.

**Comment by maurelian** [comment]
Sorry, but you now have a legit linter error [here](https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/9579/workflows/3f785f31-9365-4bd3-b562-b67755a7dc09/jobs/216589/parallel-runs/0/steps/0-104), because the code you deleted means some of the imported are no longer used.

I suggest running `yarn lint` before pushing. 

Thanks for your persistence!

**Review by smartcontracts — APPROVED** [review]

**Comment by protocolwhisper** [comment]
There's any change I need to make so it can be merged?

**Review by maurelian — APPROVED** [review]

**Comment by protocolwhisper** [comment]
Awesome, No worries I'm seeking to contribute more :100:

**Pull request merged**